kalite/distributed/static/js/distributed/exercises/perseus-helpers.js

```diff
@@ -39,8 +37,23 @@ window.Exercises = {
     RelatedVideos: {
         render: function() {}
     },
-    incompleteStack: [0]
-};
+    incompleteStack: [0],
+    proxyTrigger: function () {
```

Comment:
This is a bit opaque, and perhaps deserves at least a comment (especially if we're going to get it merged upstream).